### ISSUE TITLE
New version notification

### ISSUE THREAD

**Issue by FreeTrade** [HOdlcoin/coinomi-android#6]
New version notification is probably for the original coinomi. Best to disable this

**Comment by amit177** [comment]
can you attach a picture where you see that?

**Comment by FreeTrade** [comment]
I think this is an Android Play Store notification trying to get updated to the latest official Coinomi version.

I think a quick dirty fix is to change our version number from Current Version from 1.6.3 to 10.6.3 - should sort it out. I can't find that version number in the source though - is it in a build file maybe?

**Comment by amit177** [comment]
https://github.com/HOdlcoin/coinomi-android/blob/master/wallet/build.gradle#L12

I think that it happens because the application id is the same, should we try to change it?

**Comment by FreeTrade** [comment]
yes, alternatively try changing the app id if it is a small job - just want something quick and fast that'll work for the beta and we'll address the problem properly for the app store release.
